Document the jurisprudence of the "fake news law" (#45)
ambanum/disinfo.quaidorsay.fr: _pages/encyclopedia/4-reaction/1-case-studies.md

### The French law against information manipulation

> In France, a law against information manipulation and the **intentional** spread of disinformation was officially adopted on November 20, 2018, nine months after its proposal and enacted on December 22, 2018.

Reviewer: Non ! Les décrets [ont été publiés](https://www.legifrance.gouv.fr/affichLoiPubliee.do?idDocument=JORFDOLE000037151987&type=echeancier&typeLoi=&legislature=15) le 10 avril.

It focuses on massive and rapid spreads of disinformation on social media and more widely digital tools, including foreign state-owned media outlets, and was meant to be especially efficient in electoral contexts.

During said times, the law compels platforms to be more transparent by, for instance, reporting sponsored content as well as the name and invested resources of sponsors. The biggest, most used platforms are to have a legal representative in France and publish their algorithms as open data.

> "Fake news" are also formally defined in order to allow for judiciary action. This definition includes deliberate, massive and artificial public diffusion, capacity of disturbance (of public peace or elections), and undeniability of the falsehood or misleading nature of the information being spread.

Reviewer: “capacity for” ou plutôt “capability of”, “capacity of” signifie “capacité” dans le sens de la contenance.

```suggestion
"Fake news" are also formally defined in order to allow for judiciary action. This definition includes deliberate, massive and artificial public diffusion, capability for disruption (of public peace, elections…), and undeniability of the falsehood or misleading nature of the information being spread.
```

Outside of election times, platforms ought to cooperate, as well as build and implement open measures to fight disinformation.
The French Superior Council of the Audiovisual (CSA) has been given the authority to hinder or interrupt the broadcasting of foreign state-owned (or state-influenced) TV-services in cases where fundamental national interests are at stake.

#### [Jurisprudence](https://www.dalloz-actualite.fr/flash/loi-fake-news-premiere-application-du-refere)

> On May 10, 2019 two French elected representatives have asked for the takedown of a [tweet](https://twitter.com/ccastaner/status/1123664392011304961) from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were [only seeking refuge](https://www.lemonde.fr/politique/article/2019/05/02/incidents-a-la-pitie-salpetriere-castaner-contredit-par-une-video-l-opposition-demande-des-explications_5457544_823448.html).

Reviewer: ```suggestion
On May 10, 2019 two French elected representatives asked for the takedown of a [tweet](https://twitter.com/ccastaner/status/1123664392011304961) by the minister of Interior claiming that a hospital in Paris had been attacked during a demonstration on May 1. However, it was shortly after [proven](https://www.lemonde.fr/politique/article/2019/05/02/incidents-a-la-pitie-salpetriere-castaner-contredit-par-une-video-l-opposition-demande-des-explications_5457544_823448.html) that these people were only seeking refuge.
```

> Based on the 2018 law against information manipulation, they brought the case to court on May 10 and obtained a hearing on the 16th, but  the decision to clear both the Minister and Twitter was given on May 17. 

Reviewer: From what I understand Twitter was not liable, and thus did not have to be “cleared” of anything. The ruling could have resulted in an obligation for Twitter to delete that tweet, but it did not.

> According to the judgement, the law does not comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 

Reviewer: ```suggestion
According to the ruling, the case was out of the scope of the law, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 
```

The content should be:
- **"false or misleading allegations or attributions"** - however the content of the tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;

> - of a nature **"able to alter the sincerity of the next election"** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own opinion. 

Reviewer: ```suggestion
- of a nature **"able to alter the sincerity of the next election"** - but the strong negative reactions to the tweet were said to be enough to allow citizens to build their own opinion. 
```

Its dissemination should be:
- **"deliberate, artificial or automated"** - but this was not demonstrated ;
- **"massive"** ;
- **"public through an online communication service"**.

> Also, Twitter France was not held responsible as it claimed that it only monetized the information network of Twitter International, who thus was actually responsible for data use and processing. 

Reviewer: I am not sure it was “not held responsible”, but rather that it argued (and won) that it should not be cited at all, but rather Twitter International.

### The French law against information manipulation

> In France, a law against information manipulation and the **intentional** spread of disinformation was officially adopted on November 20, 2018, nine months after its proposal and the start of the accelereated procedure on 26 March 2018.

Reviewer: No, the decree publication date is the date on which the law is actually enacted, not the beginning of the procedure.

Outside of election times, platforms ought to cooperate, as well as build and implement open measures to fight disinformation.
The French Superior Council of the Audiovisual (CSA) has been given the authority to hinder or interrupt the broadcasting of foreign state-owned (or state-influenced) TV-services in cases where fundamental national interests are at stake.

#### Jurisprudence

> On May 10, 2019 two French politicians (a European deputy and a senator) have asked for the **takedown of a Tweet** from the Minister of Interior claiming that a hospital in Paris was attacked during a demonstration on May 1 by a group Yellow Vests. However, it has been proven that these people were seeking refuge and had no intent to harm the hospital whatsoever.

Reviewer: - Unnecessary details on “politicians”.  “elected representatives” should be enough.
- Don't use bold.
- Add link to tweet and to proof.
- “only seeking refuge”, remove “had no intent…”

> Based on the 2018 law against information manipulation, they went to court on May 16 but  the **decision to clear both the Minister and Twitter** was given on May 17. 

Reviewer: 10, not 16.

[Source](https://www.dalloz-actualite.fr/flash/loi-fake-news-premiere-application-du-refere)

Don't use bold.

Author: Non, c'est bien le bon déroulement :)
- cour saisie le 10
- audience le 16
- décision le 17

Reviewer: Ok mais il faut documenter le délai de 6 jours entre la saisie et l'audience ! Dans l'idée c'est 48h…

> It was judged that said law was to restrictive to comprehend the case within its scope, as **not [all criteria](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000037850595&dateTexte=&categorieLien=cid) were met**. 

Reviewer: “too restrictive” is a judgement, and it is not what was ruled. I'm not sure I understand your point here. The law was applied.

The content should be:

> - **false or misleading allegations or attributions** - however the content of the Tweet relied on actual events (an intrusion in the hospital) and deemed only an exaggeration;

Reviewer: Use quotes here!!

> - of a nature **able to alter the sincerity of the next election** - but the strong negative reactions to the Tweet were said to be enough to allow citizens to build their own enlightend opinion without being influenced. 

Reviewer: No capital to “tweet”.
“their own enlightend opinion without being influenced” → “their own opinion”

Its dissemination should be:

> - **deliberate, artificial or automated** - but no third party was paid to so accentuate the dissemination ;

Reviewer: No, this was simply not demonstrated.

[Source](https://twitter.com/PJanuel/status/1131522202048897024).

- **massive** ;

> - **public through an online communication service**.

Reviewer: Wouldn't these criteria be more properly listed above?

Author: Completed

> Also, Twitter France was acquitted as it claimed that it only monetized the information network of Twitter International, who thus was actually responsible for data use and processing. 

Reviewer: “acquitted”? Of what was it accused?

> Moreover, the two initiators were ordered to bear the costs of the proceedings.

Reviewer: This is standard practice when the case is lost, I don't see the added value of adding this precision.
